zivid_samples/src/sample_capture_with_settings_from_yml.cpp

```diff
@@ -0,0 +1,59 @@
+#include <zivid_camera/SettingsAcquisitionConfig.h>
+#include <zivid_camera/SettingsConfig.h>
+#include <zivid_camera/Capture.h>
+#include <zivid_camera/LoadSettingsFromFile.h>
+#include <dynamic_reconfigure/Reconfigure.h>
+#include <dynamic_reconfigure/client.h>
+#include <sensor_msgs/PointCloud2.h>
+#include <ros/ros.h>
+#include <ros/package.h>
+
+#define CHECK(cmd)                                                                                                     \
+  do                                                                                                                   \
+  {                                                                                                                    \
+    if (!cmd)                                                                                                          \
+    {                                                                                                                  \
+      throw std::runtime_error{ "\"" #cmd "\" failed!" };                                                              \
+    }                                                                                                                  \
+  } while (false)
+
+namespace
+{
+const ros::Duration default_wait_duration{ 30 };
+
+void capture()
+{
+  ROS_INFO("Calling capture service");
+  zivid_camera::Capture capture;
+  CHECK(ros::service::call("/zivid_camera/capture", capture));
+}
+
+}  // namespace
+
+int main(int argc, char** argv)
+{
+  ros::init(argc, argv, "sample_capture_cpp");
```

Comment:
can you update this to be correct for this sample?

Reply:
done